inginious/frontend/flask/mongo_sessions.py

```diff
@@ -67,31 +66,26 @@ def _get_signer(self, app):
                       key_derivation='hmac')
 
     def open_session(self, app, request):
-        # Check for cookieless session in the path
-        path_session = request.args.get('session_id')
+        # Check for LTI session in the path
+        lti_session = request.args.get('session_id')
 
-        # Check if currently accessed URL is LTI launch page
+        # Check if currently accessed URL is LTI launch pages
         try:
             # request.url_rule is not set yet here.
             endpoint, _ = app.create_url_adapter(request).match()
-            is_lti_launch = endpoint == LTILaunchPage.endpoint
+            is_lti_launch = endpoint in [LTIOIDCLoginPage.endpoint, LTILaunchPage.endpoint]
         except HTTPException:
             is_lti_launch = False
 
-        if path_session:  # Cookieless session
-            cookieless = True
-            sid = path_session
-        elif is_lti_launch:
-            cookieless = True
-            sid = None
-        else:
-            cookieless = False
-            sid = request.cookies.get(self.get_cookie_name(app))
+        if lti_session or is_lti_launch:
+            return None
```

Comment:
My understanding here is that, before, we returned the result of `self.session_class(sid=None)` while here we return directly `None`. Is that equivalent ?